src/main/java/lmr/randomizer/Settings.java

```diff
@@ -64,8 +65,8 @@
     private List<String> enabledGlitches = new ArrayList<>();
     private List<String> enabledDamageBoosts = new ArrayList<>();
 
-    private List<String> possibleRandomizedItems = Arrays.asList("Holy Grail", "Hand Scanner", "reader.exe",
-            "Hermes' Boots", "Grapple Claw", "Feather", "Isis' Pendant", "Bronze Mirror", "mirai.exe", "bunemon.exe", "Ring");
+    private static final String[] possibleRandomizedItems = {"Holy Grail", "Hand Scanner", "reader.exe",
```

Comment:
Oh, this was pointless, Collections.unmodifiableList was what I wanted (except I definitely don't want that boilerplate)

Reply:
Of course it's a harmless change and a bit more compact but bleh, I did this because it was easier than checking whether it's sneakily modified (it's not of course) and it wouldn't've told me any such thing.